salt/_compat.py

```diff
@@ -1,248 +1,12 @@
 """
 Salt compatibility code
 """
-# pylint: disable=import-error,unused-import,invalid-name,W0231,W0233
-
-
-import binascii
-import logging
+# pylint: disable=unused-import
 import sys
 
-from salt.exceptions import SaltException
-from salt.ext.six import binary_type, integer_types, string_types, text_type
-from salt.ext.six.moves import StringIO, cStringIO
-
-log = logging.getLogger(__name__)
-
-try:
-    # Python >2.5
-    import xml.etree.cElementTree as ElementTree
```

Comment:
Why are we removing the C implementation of the library as an option to users? Could we continue to allow this import in case users want to use the speed of the C library?

Reply:
Python's element tree **is** the current C implementation :)

Reply:
https://docs.python.org/3.5/library/xml.etree.elementtree.html

Reply:
Look at the top.

Reply:
Thats great news! thanks :)